### PR TITLE
Add optional Dropout to model.

### PR DESCRIPTION
**Pull request by ab7289**
Added in Dropout to the ResidualBlock module, as well as testing with that.

### CODE REVIEW

File: src/model.py [hunk 1]
```diff
@@ -86,8 +90,8 @@ def __init__(self, architecture: List[Tuple[int,int]], stem_config: Optional[Ste
 
         self.body = nn.Sequential()
         for idx, layer_def in enumerate(architecture):
```
Comment (os0000os) [inline src/model.py]: if "architecture" includes multiple "blocks", and each "block" includes multiple "layers", then perhaps it'd be easier to read if we rename "layer_def" to "block_def"?

**Review by os0000os — DISMISSED** [review]
What does "_48m" stand for in the filename "alex_48m_dropout"? Just curious.

**Comment by ab7289** [comment]
> What does "_48m" stand for in the filename "alex_48m_dropout"? Just curious.

Just stands for 4.8m parameters.

File: src/model.py (hunk 1 above)
Comment (ab7289) [reply src/model.py]: good idea

**Review by os0000os — APPROVED** [review]

**Pull request merged**